bdk-python/tests/test_live_wallet.py

```diff
@@ -64,6 +64,11 @@ def test_broadcast_transaction(self):
         walletDidSign = wallet.sign(psbt)
         self.assertTrue(walletDidSign)
         tx = psbt.extract_tx()
+        print(f"Transaction Id: {tx.txid}")
+        fee = wallet.calculate_fee(tx)
+        print(f"Transaction Fee: {fee}")
+        fee_rate = wallet.calculate_fee_rate(tx)
+        print(f"Transaction Fee Rate: {fee_rate} sat/vB")
```

Comment:
I don't think you're printing the value here but just the object. Me thinks it needs `fee_rate.as_sat_per_vb()`.

Reply:
Definitely, miss by me, updated-